### PR TITLE
Created doc: Architecture overview

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Each service has its executable binaries and processes launched from the binaries. You can deploy a Nebula Graph cluster on a single machine or multiple machines using these binaries.
+
+The following figure shows the logical architecture of a typical Nebula Graph cluster.
+
+![Nebula Graph architecture](https://docs-cdn.nebula-graph.com.cn/docs-2.0/1.introduction/2.nebula-graph-architecture/nebula-graph-architecture2.png "Nebula Graph architecture")
+
+## The Meta Service
+
+The Meta Service in the Nebula Graph architecture is supported by a cluster composed of nebula-metad processes. It is responsible for metadata storage and processing and tasks such as CRUD operations on schema structures, cluster administration, and user privilege management.
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:13]: . It is responsible for metadata storage, such as  schema operations, cluster administration, and user privilege.

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+For details on the Meta Service, see [Meta Service](2.meta-service.md).
+
+## The Graph Service and the Storage Service
+
+Nebula Graph supports separated computing and storage:
+
+* The Graph Service is responsible for computing.
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:21]: The Graph Service is responsible for querying.

**Review by whitewum — COMMENTED** [review]
some errors.

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Architecture overview
+
+A Nebula Graph cluster consists of three services: the Graph Service, the Storage Service, and the Meta Service.
+
+Each service has its executable binaries and processes launched from the binaries. You can deploy a Nebula Graph cluster on a single machine or multiple machines using these binaries.
+
+The following figure shows the logical architecture of a typical Nebula Graph cluster.
```
Comment (Amber1990Zhang) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:7]: The following figure shows the architecture of a typical Nebula Graph cluster.

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md (hunk 1 above)
Comment (Amber1990Zhang) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:13]: The Meta Service in Nebula Graph is supported by the nebula-metad process.

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## The Meta Service
+
+The Meta Service in the Nebula Graph architecture is supported by a cluster composed of nebula-metad processes. It is responsible for metadata storage and processing and tasks such as CRUD operations on schema structures, cluster administration, and user privilege management.
+
+For details on the Meta Service, see [Meta Service](2.meta-service.md).
+
+## The Graph Service and the Storage Service
+
+Nebula Graph supports separated computing and storage:
```
Comment (Amber1990Zhang) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md]: Disaggregated storage and compute architecture

File: docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## The Meta Service
+
+The Meta Service in the Nebula Graph architecture is supported by a cluster composed of nebula-metad processes. It is responsible for metadata management, such as schema operations, cluster administration, and user privilege management.
+
+For details on the Meta Service, see [Meta Service](2.meta-service.md).
+
+## The Graph Service and the Storage Service
+
+Nebula Graph supports separated computing and storage. The Graph Service is responsible for querying. The Storage Service is responsible for storage. And they run on different processes, i.e., nebula-graphd and nebula-storaged. The benefits of separated computing and storage are as follows:
```
Comment (Amber1990Zhang) [inline docs-2.0/1.introduction/3.nebula-graph-architecture/1.architecture-overview.md:19]: Disaggregated storage and compute architecture

**Review by whitewum — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**